### ISSUE TITLE
Cannot run 2 Zowe varying by IP with DVIPA, validation fails

### ISSUE THREAD

**Issue by 1000TurquoisePogs** [zowe/zowe-install-packaging#2943]
**Describe the bug**
Users cannot run Zowe using dvipa when 2 instances of zowe are using the same port but on different IPs. The config allows this, but the validation fails. You get an error like:

```
Error 0: Port 7553 is already in use by process (ZWE2AD 0004F17C Listen)
2022-07-04 16:30:49 <ZWELS:33620822> ZWESVUSR ERROR (prepare-instance.sh,validat_components:29) Error 0: Port 7553 is already in use by process (ZWE2AD 0004F17C Listen)
```

**Expected behavior**
Both Zowe should run, because they are on different IPs so they can use the same port number.

**Additional context**
This appears to happen due to the use of netstat within https://github.com/zowe/zowe-install-packaging/blob/89e6b674e81363656904251f605b26835eea8856/bin/utils/network-utils.sh#L157 not having IP information, just port information.

**Comment by 1000TurquoisePogs** [comment]
It appears on zos, the -B flag would solve it.

```
> onetstat -B 0.0.0.0+22 -c SERVER
MVS TCP/IP NETSTAT CS V2R5       TCPIP Name: TCPIP           07:56:28
User Id  Conn     State
-------  ----     -----
SSHD3    0000004A Listen
  Local Socket:   0.0.0.0..22
  Foreign Socket: 0.0.0.0..0
> onetstat -B 192.168.21.23+22 -c SERVER
MVS TCP/IP NETSTAT CS V2R5       TCPIP Name: TCPIP           07:56:36
User Id  Conn     State
-------  ----     -----
```

**Comment by jackjia-ibm** [comment]
Could you provide more information on:

- how DVIPA is configured
- how `haInstances` are defined in zowe.yaml
- how zowe is started

We did test v2 on internal Sysplex and we didn't see this issue. So there are couldn't something else.

Zowe configuration doesn't really touch the bind address yet. If we do, the netstat check will be very complicated. So I'm also happy to remove the port checks due to the potential complexity.

**Comment by jackjia-ibm** [comment]
A little more chat with James and found the user may try to start 2 Zowe on same LPAR on same port. This won't work without customization since by default many components are bound to 0.0.0.0 like this https://github.com/zowe/api-layer/blob/v2.x.x/discovery-package/src/main/resources/bin/start.sh#L133. Changing the bind address may also affect the $ZWE_DISCOVERY_SERVICES_LIST definition. So sounds like this is an issue more than just port check with netstate.

**Comment by 1000TurquoisePogs** [comment]
User had told us that they previously had 2 unrelated zowe instances and that prior to the port checking function being called, they both worked fine. But upon upgrading to a version of zowe that did the port check, now they cant start 2 at once. It's not an HA scenario. I will write back here when I get more info on their dvipa config.

**Comment by 1000TurquoisePogs** [comment]
Implemented in 1.28.2 and 2.5
